### ISSUE TITLE
How to keep track over time and act according to GDPR?

### ISSUE THREAD

**Issue by jouni-kantola** [bromso/connect-the-dots#3]
What type of information would need to be saved to provide valuable insights? In what way would that information be made sure to adhere to GDPR?

**Comment by venomtm** [comment]
I don't think it's necessary to store any information regarding individual students at all. You should be able to act based on information in an anonymized format like: "2 out of 30 students in this class...".

**Comment by jouni-kantola** [comment]
@venomtm: But wouldn't that be like a performance metrics without any actionable insights?

**Comment by venomtm** [comment]
@jouni-kantola: I don't think so. It would of course be harder, but the teachers would know on paper (app) that something is wrong. They would then have to pay more attention to individual students.
